samples/with-k8s-ref/theia-nodejs.yaml

```diff
@@ -0,0 +1,81 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: theia-nodejs
+spec:
+  started: true
+  template:
+    projects:
+      - name: project
+        git:
+          remotes:
+            origin: "https://github.com/che-samples/web-nodejs-sample.git"
+    components:
+      - name: che-theia
+        plugin:
+          kubernetes:
+            name: theia-next
+            namespace: devworkspace-plugins
+      - name: machine-exec
+        plugin:
+          kubernetes:
+            name: machine-exec
```

Comment:
```
      - name: che-theia
        plugin:
          kubernetes:
            name: theia-next
            namespace: devworkspace-plugins
```
already includes machine-exec, do we need to specify it here again?

Reply:
I'll fix up the samples :) 

I mostly copy-pasted from scratch files I had for testing so they're sloppy.

Reply:
Currently, theia-next.yaml doesn't include `machine-exec` as far as I know -- if it does applying this DevWorkspace should result in failure to start.